lib/util/conflicter.js

```diff
@@ -165,7 +171,31 @@ Conflicter.prototype._ask = function (file, cb) {
     }
 
     if (result.action === 'diff') {
-      this.adapter.diff(fs.readFileSync(file.path, 'utf8'), file.contents.toString());
+      var existingHeader = readChunk.sync(file.path, 0, 512);
+
+      if (istextorbinary.isBinarySync(undefined, existingHeader) || istextorbinary.isBinarySync(undefined, file.contents)) {
+        var existingStat = fs.statSync(file.path);
+
+        var table = new Table({
+          head: ['', 'Existing', 'Replacement', 'Diff']
+        });
+
+        table.push(
+          ['Size',
+            prettyBytes(existingStat.size),
+            prettyBytes(file.contents.length),
+            ((existingStat.size > file.contents.length) ? '-' : '+') + prettyBytes(Math.abs(existingStat.size - file.contents.length))],
```

Comment:
This line is too long and very complicated. Move it to an `if` statement on another line.